.gitignore

```diff
@@ -0,0 +1,41 @@
+## ./augogen.sh resulting files:
+aclocal.m4
+config.h.in
+configure
+Makefile.in
+autom4te.cache/
+
+m4/libtool.m4
+m4/ltversion.m4
+m4/lt~obsolete.m4
+m4/ltoptions.m4
+m4/ltsugar.m4
+
+# This file is modified during autogen.sh, not sure if it should be added to .gitignore
+build-aux/ltmain.sh
```

Comment:
This is most likely a left over. Code can be sucessfully compiled without this file. We should remove it.

Reply:
Interesting, I remember testing it in a fresh environment (clonned repo again) and it didn't work.
But maybe I got confused and tried it in the wrong folder. I can't test it right now, but maybe tomorrow I'll try it again.